pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -669,6 +670,10 @@ self: super: {
     dontBuild = true;
   });
 
+  vim-mediawiki-editor = super.vim-mediawiki-editor.overrideAttrs (old: {
+    dependencies = with self; [ python310Packages.mwclient ];
```

this is probably what you want
```suggestion
    passthru.python3Dependencies = [ python3.pkgs.mwclient ];
```

---

Yep, nice catch. Added in the latest push.